authors/apps/articles/renderers.py

```diff
@@ -10,8 +10,14 @@ def render(self, data, media_type=None, renderer_context=None):
         if isinstance(data, ReturnList):
             return self.render_articles(data)
 
+        if data is None:
+            return ''
+
         errors = data.get('errors', None)
 
+
+        article = data.get('article', None)
```

Comment:
Hey @Manorlds-Eaglespark, I was wondering, what does this line do? I don't see `article` being rendered anywhere in the return statement. Also on, lines 13-14, why is an empty string returned? Do you get errors when these lines are removed?